src/helpers/url.js

```diff
@@ -38,7 +38,7 @@ const getCallbackBaseUrl = () => {
 }
 
 const getWopiSrc = (fileId) => {
-	// WOPISrc - URL that loolwsd will access (ie. pointing to ownCloud)
+	// WOPISrc - URL that Collabora will use to access Nextcloud
```

Comment:
@blizzz Sneaked this in from your comment in https://github.com/nextcloud/richdocuments/pull/3575 😁